mne/io/ctf/ctf.py

```diff
@@ -33,7 +33,7 @@
 @fill_doc
 def read_raw_ctf(
     directory, system_clock="truncate", preload=False, clean_names=False, verbose=None
-):
+) -> BaseRaw:
```

Comment:
Why? It returns RawCTF

Reply:
It was an attempt to fix the Sphinx error but I found a better solution